package.json

```diff
@@ -462,6 +462,12 @@
             "description": "%param.flashBaudRate%",
             "scope": "resource"
           },
+          "idf.monitorBaudRate": {
+            "type": "string",
+            "default": "460800",
```

Comment:
I agree with @JPZV . the extension just updated in my vscode and this caused the monitor to read garbage to me. I imagine this will happen for a lot of people. so.... everyone, just open your VSCode User Settings (JSON) (Ctrl+P, "settings json"), then add
```javascript
// Code/User/settings.json
{
    ...
    "idf.monitorBaudRate": "115200",
    ...
}
```

Reply:
I wouldn't have known how to approach this if I hadn't seen the "update notification" and read the changelog. Might have reached for PuTTY to sanity-check or maybe a logic analyzer (pulseview + $5 fx2lp-based/"8ch 24M" device) to make sure the data is timed right.